html/cloudflare-1000-error.htm

```diff
@@ -198,7 +198,18 @@ <h1>Internal Error</h1>
       </ul>
     </footer>
 
-    <img src="https://volley.artsy.net/cloudflareError.png?cloudflareErrorType=1000&rayID=::RAY_ID::&clientIP=::CLIENT_IP::" />
+    <div id="error">
```

Comment:
are we displaying these? maybe we want to hide, assuming it will render a duplicate of the other error message.

Reply:
Would be good to see a link/screenshot to the updated file.

Reply:
it's past the footer so I can't see it, but yeah some CSS to hide may be good

Reply:
@eessex re: 

> Would be good to see a link/screenshot to the updated file.

Not sure which file you mean here

Reply:
The png file returned by volley is a 1x1 pixel

Reply:
Re link, I meant a link to s3 or whatever you tested the preview with.

Reply:
Ah right - I just uploaded the `cloudflare-500-error.htm` file to `html/cloudflare-500-error.htm-test` in S3 then tested it in the "preview" function in Cloudflare (but deleted it now)